eclipse/mars/gcp-eclipse-mars.tpd

```diff
@@ -27,7 +27,7 @@ location "http://download.eclipse.org/releases/mars/" {
 	org.eclipse.m2e.wtp.sdk.feature.feature.group
 	org.eclipse.mylyn.commons.feature.group
 	org.eclipse.jpt.jpa.feature.feature.group
-	org.eclipse.datatools.sdk.feature.feature.group
+	//org.eclipse.datatools.sdk.feature.feature.group
```

Comment:
Why not removing entirely (in all three `.tpd` files)? Otherwise, a comment would help.

Reply:
sure.  since they're not in the generated `.target` files, I'll just tweak them here directly.